src/rmp/test/cpp/TestAbc.cc

```diff
@@ -457,7 +462,7 @@ TEST_F(AbcTest, InsertingMappedLogicCutDoesNotThrow)
   sta::dbNetwork* network = sta_->getDbNetwork();
   sta::Vertex* flop_input_vertex = nullptr;
   for (sta::Vertex* vertex : *sta_->endpoints()) {
-    if (std::string(vertex->name(network)) == "_32989_/D") {
+    if (std::string(vertex->name(network)) == "_33122_/D") {
```

Comment:
Why does this change?  It seems brittle.

Reply:
It's just a different path. I changed it because it's the critical endpoint.